packages/discord.js/src/structures/User.js

```diff
@@ -183,9 +182,9 @@ class User extends Base {
    * @param {ImageURLOptions} [options={}] Options for the Image URL
    * @returns {?string}
    */
-  bannerURL({ format, size, dynamic } = {}) {
+  bannerURL(options = {}) {
     if (!this.banner) return this.banner;
-    return this.client.rest.cdn.Banner(this.id, this.banner, format, size, dynamic);
+    return this.client.rest.cdn.Banner(this.id, this.banner, options);
```

Comment:
You can do the same thing you did on the other ones with `this.banner && …`

Reply:
Done this!